setup.py

```diff
@@ -35,5 +35,5 @@
         'License :: OSI Approved :: Apache Software License',
         'Operating System :: OS Independent',
     ],
-    python_requires='>=3.7',
+    python_requires='>=3.6',
```

Comment:
I think we'll need to add the `dataclasses` back-compat package to `install_requires` if we want to support Python 3.6.